src/v0/destinations/customerio/util.js

```diff
@@ -295,6 +297,18 @@ const defaultResponseBuilder = (message, evName, userId, evType, destination, me
   return { rawPayload, endpoint, requestConfig };
 };
 
+const validateConfigFields = destination => {
+  const { Config } = destination;
+  if (!isDefinedAndNotNull(Config)) {
+    throw new ConfigurationError(" No Configs Found");
```

Comment:
why is this ConfigurationError?
when this case will happen?
is there any other places where we are handling this?